applications/zigbee_weather_station/src/weather_station.h

```diff
@@ -9,6 +9,44 @@
 
 #include <zcl/zb_zcl_temp_measurement_addons.h>
 
+#include "sensor.h"
+
+/* Zigbee Cluster Library 4.4.2.2.1.1: MeasuredValue = 100x temperature in degrees Celsius */
+#define ZCL_TEMPERATURE_MEMASUREMENT_MEASURED_VALUE_MULTIPLIER 100
```

Comment:
There is a typo in `.._MEMASUREMENT_..`. It applies to all macros representing multipliers.